test/spec/vis/layers-factory.spec.js

```diff
@@ -106,20 +109,22 @@ describe('vis/layers-factory', function () {
           spyOn(LayersFactory, 'isHttps').and.returnValue(true);
         });
 
-        it("should not convert '" + httpsUrlTemplate + "'", function () {
-          var layerModel = layersFactory.createLayer('tiled', {
-            urlTemplate: httpsUrlTemplate
+        describe('is high resolution screen', function () {
```

Comment:
where are we mocking high resolution screen?

Reply:
my fault, this line is no longer needed 👍